distributionviewer/core/static/js/app/components/containers/chart-list-container.js

```diff
@@ -12,7 +12,7 @@ export class ChartListContainer extends React.Component {
     this.state = {numCharts: 0};
   }
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
Yeah, eslint was complaining about `this.setState()` being used inside `componentDidMount`. I think it's because changing the state after the component mounts triggers a re-render.